tests/integration/pilot/main_test.go

```diff
@@ -38,7 +38,9 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite("pilot_test", m).
 		RequireSingleCluster().
-		SetupOnEnv(environment.Kube, istio.Setup(&i, nil)).
+		SetupOnEnv(environment.Kube, istio.Setup(&i, func(cfg *istio.Config) {
+			cfg.Values["global.outboundTrafficPolicy.mode"] = "ALLOW_ANY"
```

Comment:
What is this needed for?

Reply:
it is needed when using `ForwardEcho` to connection pod IP. removed now since using `client.Call` instead.